### ISSUE TITLE
Routing sometimes gives 0 elevation

### ISSUE THREAD

**Issue by HarelM** [IsraelHikingMap/Site#1647]
## Bug
When calling the following API call sometimes the server returns 0 for all elevation:
```
https://israelhiking.osm.org.il/api/routing?from=32.937475,35.488958&to=32.935576452935365,35.48835922129717&type=Hike
```

Bad response:
```
{"type":"FeatureCollection","features":[{"type":"Feature","bbox":[35.487612,32.935563,35.488958,32.937665],"geometry":{"type":"LineString","coordinates":[[35.488958,32.937475,0.0],[35.488836,32.937593,0.0],[35.488717,32.937665,0.0],[35.488599,32.937575,0.0],[35.488546,32.93735,0.0],[35.488396,32.937143,0.0],[35.488235,32.937044,0.0],[35.487763,32.936864,0.0],[35.487612,32.936539,0.0],[35.487706,32.936266,0.0],[35.487738,32.936023,0.0],[35.487867,32.935789,0.0],[35.488012,32.935753,0.0],[35.48835,32.935563,0.0]]},"properties":{"details":{"road_class":[["0","11","track"],["11","13","cycleway"]],"track_type":[["0","13","missing"]]},"Name":"Routing from 32.937475,35.488958 to 32.935576452935365,35.48835922129717 profile type: Foot","Creator":"IsraelHikingMap"}}],"bbox":null}
```
Probably related to #1642, I need to investigate further...

Also this call:
```
http://localhost:5000/api/elevation?points=32.9356,35.4883%7C32.9355,35.4884%7C32.9352,35.4888%7C32.9351,35.4889%7C32.9349,35.4890%7C32.9347,35.4891%7C32.9346,35.4892%7C32.9344,35.4892%7C32.9341,35.4893%7C32.9338,35.4893%7C32.9337,35.4891%7C32.9337,35.4886%7C32.9337,35.4886%7C32.9336,35.4880%7C32.9338,35.4875%7C32.9336,35.4868%7C32.9334,35.4856%7C32.9334,35.4852%7C32.9334,35.4849%7C32.9335,35.4844%7C32.9336,35.4842%7C32.9340,35.4837%7C32.9342,35.4835%7C32.9347,35.4833%7C32.9350,35.4833%7C32.9357,35.4832%7C32.9360,35.4830%7C32.9366,35.4825%7C32.9368,35.4823%7C32.9371,35.4822%7C32.9375,35.4821%7C32.9380,35.4820%7C32.9382,35.4818%7C32.9384,35.4816%7C32.9388,35.4814%7C32.9390,35.4813%7C32.9393,35.4811%7C32.9397,35.4811%7C32.9400,35.4812%7C32.9408,35.4815%7C32.9411,35.4816%7C32.9413,35.4815%7C32.9415,35.4814%7C32.9415,35.4812%7C32.9418,35.4812%7C32.9422,35.4810%7C32.9424,35.4810%7C32.9427,35.4811%7C32.9428,35.4811%7C32.9430,35.4809%7C32.9431,35.4807%7C32.9435,35.4806%7C32.9436,35.4804%7C32.9438,35.4803%7C32.9442,35.4801%7C32.9443,35.4801%7C32.9449,35.4800%7C32.9450,35.4800%7C32.9454,35.4801%7C32.9456,35.4802%7C32.9460,35.4805%7C32.9462,35.4805%7C32.9464,35.4805%7C32.9464,35.4807%7C32.9465,35.4808%7C32.9465,35.4809%7C32.9469,35.4809%7C32.9472,35.4809%7C32.9476,35.4810%7C32.9481,35.4811%7C32.9484,35.4811%7C32.9485,35.4810%7C32.9485,35.4808%7C32.9493,35.4807%7C32.9494,35.4807%7C32.9500,35.4809%7C32.9502,35.4810%7C32.9502,35.4809
```

**Comment by HarelM** [comment]
According to server logs this is due to an exception that happens when allocating memory.
One of the problems is that the elevation is reassigned for every routing on all points.
Another might be at the elevation service itself - as memory release there is not happening as it should be, maybe...

**Comment by HarelM** [comment]
Closed via d9e8dd4
